federation/pkg/federation-controller/configmap/configmap_controller.go

```diff
@@ -44,6 +45,11 @@ import (
 
 const (
 	allClustersKey = "ALL_CLUSTERS"
+	ControllerName = "configmaps"
```

Comment:
This is controller name. Since we have a controller for each resource, we chose to use the resource name as controller name. They could be different in future.